### PR TITLE
[1.5.x][RHPAM-3654] - Wrong archetype used to generate project from provided…

### PR DESCRIPTION
**Pull request by spolti**
…… (#510)

* [RHPAM-3654] - Wrong archetype used to generate project from provided asset

Signed-off-by: spolti <fspolti@redhat.com>

* Update tests/features/kogito-builder.feature

Co-authored-by: Tristan Radisson <tristan.radisson@gmail.com>

* Update tests/features/rhpam-kogito-builder-jvm.feature

Co-authored-by: Tristan Radisson <tristan.radisson@gmail.com>

Co-authored-by: Tristan Radisson <tristan.radisson@gmail.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors guide](README.md#contributing-to-kogito-images-repository)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a testcase that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: tests/features/kogito-builder.feature [hunk 1]
```diff
@@ -78,4 +78,47 @@ Feature: kogito-builder image tests
...
+      | wait                  | 80                            |
+      | expected_status_code  | 200                           |
+      | request_method        | POST                          |
+      | content_type          | application/cloudevents+json  |
+      | request_body          | {"specversion": "1.0", "source": "behave", "type": "orderEvent", "id": "12345", "data": {"id":"f0643c68-609c-48aa-a820-5df423fa4fe0","country":"Brazil","total":10000,"description":"iPhone 12"}}|
+
+#### SpringBoot Scenarios
+
+  Scenario: Verify that the Kogito Maven archetype is generating the project and compiling it correctly when runtime is springboot
+    Given s2i build /tmp/kogito-examples from dmn-example using nightly-master and runtime-image quay.io/kiegroup/kogito-runtime-jvm:latest
```
Comment (radtriste) [inline tests/features/kogito-builder.feature]:
```suggestion
    Given s2i build /tmp/kogito-examples from dmn-example using 1.5.x and runtime-image quay.io/kiegroup/kogito-runtime-jvm:latest
```

File: tests/features/rhpam-kogito-builder-jvm.feature [hunk 2]
```diff
@@ -62,4 +62,23 @@ Feature: rhpam-kogito-builder-rhel8 feature.
...
-      | request_body    | {"Driver": {"Points": 2}, "Violation": {"Type": "speed","Actual Speed": 120,"Speed Limit": 100}} |
\ No newline at end of file
+      | request_body    | {"Driver": {"Points": 2}, "Violation": {"Type": "speed","Actual Speed": 120,"Speed Limit": 100}} |
+
+#### SpringBoot Scenarios
+
+  Scenario: Verify that the Kogito Maven archetype is generating the project and compiling it correctly when runtime is springboot
+    Given s2i build /tmp/kogito-examples from dmn-example using nightly-master and runtime-image rhpam-7/rhpam-kogito-runtime-jvm-rhel8:latest
+      | variable       | value          |
+      | KOGITO_VERSION | 2.0.0-SNAPSHOT |
```
Comment (radtriste) [inline tests/features/rhpam-kogito-builder-jvm.feature]:
```suggestion
      | KOGITO_VERSION | 1.5.0.redhat-00001 |
```

File: tests/features/kogito-builder.feature [hunk 3]
```diff
@@ -78,4 +78,47 @@ Feature: kogito-builder image tests
...
+    And check that page is served
+      | property              | value                         |
+      | port                  | 8080                          |
+      | path                  | /                             |
+      | wait                  | 80                            |
+      | expected_status_code  | 200                           |
+      | request_method        | POST                          |
+      | content_type          | application/cloudevents+json  |
+      | request_body          | {"specversion": "1.0", "source": "behave", "type": "orderEvent", "id": "12345", "data": {"id":"f0643c68-609c-48aa-a820-5df423fa4fe0","country":"Brazil","total":10000,"description":"iPhone 12"}}|
+
```
Comment (radtriste) [inline tests/features/kogito-builder.feature]:
```suggestion
```

**Review by radtriste — CHANGES_REQUESTED** [review]

File: tests/features/kogito-builder.feature (hunk 3 above)
Comment (radtriste) [reply tests/features/kogito-builder.feature]: This test should not be there. AFAIK, serverless is not in 1.5.x

File: tests/features/rhpam-kogito-builder-jvm.feature [hunk 4]
```diff
@@ -62,4 +62,23 @@ Feature: rhpam-kogito-builder-rhel8 feature.
...
       | request_method  | POST                                                                                             |
       | content_type    | application/json                                                                                 |
-      | request_body    | {"Driver": {"Points": 2}, "Violation": {"Type": "speed","Actual Speed": 120,"Speed Limit": 100}} |
\ No newline at end of file
+      | request_body    | {"Driver": {"Points": 2}, "Violation": {"Type": "speed","Actual Speed": 120,"Speed Limit": 100}} |
+
+#### SpringBoot Scenarios
+
+  Scenario: Verify that the Kogito Maven archetype is generating the project and compiling it correctly when runtime is springboot
+    Given s2i build /tmp/kogito-examples from dmn-example using nightly-master and runtime-image rhpam-7/rhpam-kogito-runtime-jvm-rhel8:latest
```
Comment (radtriste) [inline tests/features/rhpam-kogito-builder-jvm.feature]:
```suggestion
    Given s2i build /tmp/kogito-examples from dmn-example using master and runtime-image rhpam-7/rhpam-kogito-runtime-jvm-rhel8:latest
```

**Comment by radtriste** [comment]
GHAction should be solved by https://github.com/kiegroup/kogito-images/pull/515

**Review by radtriste — APPROVED** [review]

**Comment by spolti** [comment]
Thanks @radtriste

**Pull request merged**